packages/amplify-util-uibuilder/src/__tests__/cloneComponentsFromEnv.test.ts

```diff
@@ -1,37 +1,37 @@
+jest.mock('aws-sdk', () => {
```

Comment:
nit it shouldn't be necessary to call mock before import. Unless something breaks, I'd recommend all of the import statements together then all of the mock statements